api/src/pcapi/routes/backoffice/user_deletion/blueprint.py

```diff
@@ -0,0 +1,69 @@
+import logging
+
+from flask import flash
+from flask import redirect
+from flask import render_template
+from flask import url_for
+from markupsafe import Markup
+from sqlalchemy.exc import IntegrityError
+from werkzeug.exceptions import NotFound
+
+from pcapi import settings
+from pcapi.core.users import models as users_models
+from pcapi.models import db
+from pcapi.repository import atomic
+from pcapi.repository import mark_transaction_as_invalid
+from pcapi.routes.backoffice import blueprint
+from pcapi.routes.backoffice import utils
+
+from . import forms
+
+
+logger = logging.getLogger(__name__)
+
+
+@blueprint.backoffice_web.route("/delete", methods=["GET"])
+@utils.custom_login_required(redirect_to=".home")
+def get_user_deletion_form() -> str:
+    form = forms.UserDeletionForm()
+    return render_template("admin/users_deletion.html", form=form)
+
+
+@blueprint.backoffice_web.route("/delete", methods=["POST"])
+@atomic()
+@utils.custom_login_required(redirect_to=".home")
+def delete_user() -> utils.BackofficeResponse:
+    if not settings.ENABLE_USER_DELETION:
+        raise NotFound()
+
+    form = forms.UserDeletionForm()
+    if not form.validate():
+        flash(utils.build_form_error_msg(form), "warning")
+        return redirect(url_for("backoffice_web.get_user_deletion_form"), code=303)
```

Comment:
J'aurais plutôt remis directement le formulaire, avec un code 400, ainsi les erreurs seront bien affichés dedans.
Et c'est la même URL, donc un F5 rafraîchira bien comme il faut
```suggestion
        return render_template("admin/users_deletion.html", form=form), 400
```